examples/plot_unicode.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+=======================================
+Únîcödè support in documentation string
+=======================================
+
+In this example we include Unicode characters that are so required this days.✓
+"""
+
+# Code source: Óscar Nájera
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+N = 50
+x = np.linspace(-1,2,50)
```

Comment:
TODO items before merge:

* joblib must be replaced by shelve

Remarks for later (not preventing merge in nilearn)

* _path_static should be renamed path_static, as it is used in the conf.py

* the css names should be harmonized. In particular,  they should be all given w prefix, such as sphx-glr-...

* functions should not be defined in closures: _select_block, _parse_dict_structure must be moved out of the function where they are defined

* no @author in docstrings, but author as a comment in the beginning of a file,  with also the full name of the author, the original authors if any,  and "license: BSD"

* the code isn't completely pep8: there should be two empty lines between first level functions

* later: this code needs tests

Sent from my phone. Please forgive brevity and mis spelling



On Feb 17, 2015, 15:39, at 15:39, Gael Varoquaux <gael.varoquaux@normalesup.org> wrote:
>I still don't see the value of non ASCII characters in the docstring.
>You
>and I have a non ASCII character in our name. But we accept perfectly
>the
>fact that we might get email without it.
>
>Inversely, if you get an email, that says 'здравствуйте', it makes it
>really hard for you to search your mailbox with this email. It makes it
>hard to fix a typo in this mail... I am against the use of non-ASCII
>character in a whole software project that is not specificaly about
>localization.